synapse/lib/stormhttp.py

```diff
@@ -204,16 +236,17 @@ async def inetHttpConnect(self, url, headers=None, ssl_verify=True, timeout=300)
             return s_common.retnexc(e)
 
     async def _httpRequest(self, meth, url, headers=None, json=None, body=None, ssl_verify=True,
-                           params=None, timeout=300):
+                           params=None, timeout=300, allow_redirects=True, ):
         meth = await s_stormtypes.tostr(meth)
         url = await s_stormtypes.tostr(url)
         json = await s_stormtypes.toprim(json)
         body = await s_stormtypes.toprim(body)
         headers = await s_stormtypes.toprim(headers)
         params = await s_stormtypes.toprim(params)
         timeout = await s_stormtypes.toint(timeout, noneok=True)
+        allow_redirects = await s_stormtypes.tobool(allow_redirects)
 
-        kwargs = {}
+        kwargs = {'allow_redirects': allow_redirects}
```

Comment:
Do we need code to convert an exception from an iohttp redirect disallowed exception to a storm exception here somewhere?  Do we have tests that try to use a redirect and fail?

Reply:
I've added a pair of tests for failures. There is not a "redirect disallowed" exception that is raised by aiohttp..